src/screens/playing.rs

```diff
@@ -18,12 +27,27 @@ pub(super) fn plugin(app: &mut App) {
     );
 }
 
+fn playing_screen(In(id): In<Entity>, mut commands: Commands) {
+    commands
+        .entity(id)
+        .add_fn(widget::ui_root)
```

Comment:
It seems very confusing from a first glance when one should use `add_fn` and when `spawn_fn`. If this is confusing to me as a reviewer, I imagine it would be really really confusing to a newcomer.

Reply:
An explanation in the design patterns doc should mitigate that a bit, but I'm open to bikeshedding.